res/values/strings.xml

```diff
@@ -257,8 +257,8 @@
     <string name="chat_self_talk_subtitle">Messages I sent to myself</string>
     <string name="chat_contact_request">Contact request</string>
     <string name="chat_no_contact_requests">No contact requests.\n\nIf you want classic emails to appear here as contact requests, you can change the corresponding setting in the app settings.</string>
-    <string name="retry_send">Retry to send message</string>
-    <string name="send_failed">Sending message failed</string>
+    <string name="retry_send">Try to send message again?</string>
```

Comment:
i think retry_send is meant as a button or menu entry, however, as far as i know it is currently not used anyway. we should leave it alone.

Reply:
This is not a dialog. The old version was better.

Reply:
@OzancanKaratas Better is somehow subjective. _Wrong_ is the problem I was trying to solve.